### PR TITLE
Williamhe/1492709 add dot net7 preview1 support

### PR DESCRIPTION
**Pull request by william-msft**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as  https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1492709
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/Controllers/HomeController.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using System;
```
Comment (william-msft) [inline tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/Controllers/HomeController.cs:1]: Side note: this `NetCore7PreviewMvcApp` is for the new integration test: https://github.com/microsoft/Oryx/pull/1280/files#diff-41b776f6f7b713085d0c1a0c018725d6ac56517c6d23cbf7889d2d603d6ef404R28

Comment (william-msft) [inline tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/Controllers/HomeController.cs:1]: Side note: this SampleApp `NetCore7PreviewMvcApp` is needed for our new integration test: https://github.com/microsoft/Oryx/pull/1280/files#diff-41b776f6f7b713085d0c1a0c018725d6ac56517c6d23cbf7889d2d603d6ef404R28

File: tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/NetCore7PreviewMvcApp.csproj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```
Comment (cormacpayne) [inline tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/NetCore7PreviewMvcApp.csproj:4]: Change to the `TargetFramework` to point at the .NET Core 7 Preview framework

Comment (william-msft) [reply tests/SampleApps/DotNetCore/NetCore7PreviewMvcApp/NetCore7PreviewMvcApp.csproj]: Updated. Thanks!

**Review by qianz2 — APPROVED** [review]

**Comment by william-msft** [comment]
Already released

**Pull request closed without merging**